packages/frontend/src/components/workflows/actions/WorkflowActions.vue

```diff
@@ -0,0 +1,40 @@
+<script setup lang="ts">
+import Button from "primevue/button";
+import type { Workflow } from 'shared';
+import { useWorkflowStore } from "@/stores/workflows";
+import { useWorkflow } from "@/composables/useWorkflow";
+import { useSDK } from "@/plugins/sdk";
+
+interface Props {
+  workflow: Workflow;
+}
+
+const props = defineProps<Props>();
```

Comment:
I usually just inline the type in the defineProps function, since we're not re-using the Props type anyway:
```
defineProps<{
  workflow: Workflow
}>();
```